charts/osbuilder/crds/build.kairos.io_osartifacts.yaml

```diff
@@ -0,0 +1,8031 @@
+---
```

Comment:
[We tell people](https://kairos.io/docs/advanced/build/#kubernetes) to install the kairos-crds chart. I think this file should be merged with [this one](https://github.com/kairos-io/helm-charts/blob/main/charts/kairos-crds/templates/osbuilder.yaml)

Reply:
Ill update it to use the existing chart.